app/view_objects/find/results_view.rb

```diff
@@ -0,0 +1,260 @@
+module Find
+  # FIND:TODO Will be fleshed out in an upcoming ticket
+  class ResultsView
+    attr_reader :query_parameters
+
+    include ActionView::Helpers::NumberHelper
+
+    MAXIMUM_NUMBER_OF_SUBJECTS = 43
+    DISTANCE = "2".freeze
+    SUGGESTED_SEARCH_THRESHOLD = 3
+    MAXIMUM_NUMBER_OF_SUGGESTED_LINKS = 2
+    RESULTS_PER_PAGE = 10
+    MILES = "50".freeze
+
+    def initialize(query_parameters:)
+      @query_parameters = query_parameters
+    end
+
+    def query_parameters_with_defaults
+      query_parameters.except("utf8", "authenticity_token")
+        .merge(qualifications_parameters)
+        .merge(fulltime_parameters)
+        .merge(parttime_parameters)
+        .merge(hasvacancies_parameters)
+        .merge(sen_courses_parameters)
+        .merge(subject_parameters)
+    end
+
+    def courses
+      @courses ||= RecruitmentCycle.current.courses.includes(
+        :enrichments,
+        subjects: [:financial_incentive],
+        site_statuses: [:site],
+        provider: %i[recruitment_cycle ucas_preferences],
+      ).findable.page(query_parameters[:page] || 1)
```

Comment:
we should be using https://github.com/DFE-Digital/teacher-training-api/blob/main/app/services/course_search_service.rb


```
Course.findable
# findable is a little weird due to the join it uses
```

Reply:
As discussed, i'll leave that in the follow up tickets which covers this work. This is just to get the results page displaying